README.md

```diff
@@ -22,7 +22,7 @@ What you do need:
 - A Kubernetes cluster that supports Service objects of `type: LoadBalancer` ([AWS Quickstart cluster][9] or Minikube, for example)
 - `kubectl` configured with admin access to your cluster
 
-See the [deployment documentation][10] for more deployment options.
+See the [deployment documentation][10] for more deployment options if you don't meet these requirements.
 
```

Comment:
A nit, and a hill I won't die on, but the added phrase isn't necessary. Its meaning is inherent in the original. If you think it adds clarity, keep it.

Reply:
Someone on twitter asked about this specifically and so I wanted to add more color.

Reply:
The best kind of reason :-)